### PR TITLE
fix: handle AsyncApi v3 maximumValue field

### PR DESCRIPTION
**Pull request by timonback**
Swagger generates OpenApi schemas which are not 100% compatible to AsyncApi

### CODE REVIEW

File: springwolf-core/src/test/resources/asyncapi/asyncapi.json [hunk 1]
```diff
@@ -43,7 +43,6 @@
   "components": {
     "schemas": {
       "ExamplePayload": {
-        "type": "object",
```
Comment (timonback) [inline springwolf-core/src/test/resources/asyncapi/asyncapi.json:46]: The PostProcessor fixes this, which is not part of this test

**Comment by timonback** [comment]
@aerfus We stumbled upon the maximumValue field in the AsyncApi 3 spec that you also mentioned in the other PR. A review - if you find time - is highly appreciated.

**Comment by aerfus** [comment]
Hi @timonback ,
I've reviewed and have no concerns.
Do you mean my notice on exclusiveMinimum(Maximum) became a number,
not boolean?
Is so, should we rewrite range component logic?

**Review by ctasada — APPROVED** [review]
I think we can apply the same approach for https://github.com/springwolf/springwolf-core/issues/160 WDYT?

**Pull request merged**